doc/LanguageClient.txt

```diff
@@ -228,6 +228,16 @@ before timing out.
 Default: 10
 Valid options: number
 
+2.19 g:LanguageClient_hoverPreview             *g:LanguageClient_hoverPreview*
+
+Controls how hover output is displayed. Must be one of the following:
+    0 - Never use preview window, always echo hover output
+    1 - Use preview window for hover entries longer than one line (default)
+    2 - Always use preview window, never echo hover output
+
+Default: 1
+Valid options: 0 | 1 | 2
```

Comment:
For readability and extensibility, please use enum/string options, e.g., `never`, `auto`, `always`.